pkg/events/derive/derive.go

```diff
@@ -0,0 +1,78 @@
+package derive
```

Comment:
Again, I suggest to rethink if this actually needs to be a package now.
I'll suggest an alternative: add a package for the deriving functions themselves inside ebpf where the pipeline is. There you can move all the derivation functions from `events_derived.go`. That will also accomplish some needed cleanup, and that can consume the events package for the derivation interfaces.